airflow/cli/commands/standalone_command.py

```diff
@@ -159,14 +159,28 @@ def calculate_env(self):
         # Make sure we're using a local executor flavour
         executor_class, _ = ExecutorLoader.import_default_executor_cls()
         if not executor_class.is_local:
-            if "sqlite" in conf.get("database", "sql_alchemy_conn"):
-                self.print_output("standalone", "Forcing executor to SequentialExecutor")
-                env["AIRFLOW__CORE__EXECUTOR"] = executor_constants.SEQUENTIAL_EXECUTOR
-            else:
-                self.print_output("standalone", "Forcing executor to LocalExecutor")
-                env["AIRFLOW__CORE__EXECUTOR"] = executor_constants.LOCAL_EXECUTOR
+            env["AIRFLOW__CORE__EXECUTOR"] = self.get_local_executor(
+                database=conf.get("database", "sql_alchemy_conn")
+            )
+            self.print_output("standalone", f"Forcing executor to {env['AIRFLOW__CORE__EXECUTOR']}")
         return env
 
+    @classmethod
+    def get_local_executor(cls, database: str) -> str:
+        """
+        Get local executor for standalone command
+        for sqlite we need SEQUENTIAL_EXECUTOR otherwise LOCAL_EXECUTOR.
+        """
+        try:
+            return [
+                executor_name
+                for executor_name in ExecutorLoader.executors.keys()
+                if executor_name != "DaskExecutor"
```

Comment:
This looks like executor coupling, why are we excluding the Dask Executor here specifically? Could there ever be other executors that would need excluding in the same way?

Reply:
@o-nikolas My bad was testing it locally and forgot to remove it. It's not needed here.